cmd/pebble/main.go

```diff
@@ -255,53 +256,18 @@ func Parser(cli *client.Client) *flags.Parser {
 	}
 	// Add all the sub-commands of the debug command
 	for _, c := range debugCommands {
-		obj := c.builder()
```

Comment:
Why does all this code go away? Was it duplicated from above previously?

Reply:
There were minimal differences (posted below) which IMHO weren't worth to have this code duplicated. But I've reverted this refactor.
```
--- cmd.normal	2023-02-06 00:51:00.803178868 +0100
+++ cmd.debug	2023-02-06 00:51:10.103288970 +0100
@@ -2,19 +2,11 @@
 		if x, ok := obj.(clientSetter); ok {
 			x.setClient(cli)
 		}
-		if x, ok := obj.(parserSetter); ok {
-			x.setParser(parser)
-		}
-
-		cmd, err := parser.AddCommand(c.name, c.shortHelp, strings.TrimSpace(c.longHelp), obj)
+		cmd, err := debugCommand.AddCommand(c.name, c.shortHelp, strings.TrimSpace(c.longHelp), obj)
 		if err != nil {
-			logger.Panicf("cannot add command %q: %v", c.name, err)
+			logger.Panicf("cannot add debug command %q: %v", c.name, err)
 		}
 		cmd.Hidden = c.hidden
-		if c.alias != "" {
-			cmd.Aliases = append(cmd.Aliases, c.alias)
-		}
-
 		opts := cmd.Options()
 		if c.optDescs != nil && len(opts) != len(c.optDescs) {
 			logger.Panicf("wrong number of option descriptions for %s: expected %d, got %d", c.name, len(opts), len(c.optDescs))
@@ -49,6 +41,3 @@
 			arg.Name = name
 			arg.Description = desc
 		}
-		if c.extra != nil {
-			c.extra(cmd)
-		}
```